pom.xml

```diff
@@ -30,13 +30,25 @@
     </developers>
 
     <properties>
-        <jenkins.version>1.580.1</jenkins.version>
+        <jenkins.version>1.596.1</jenkins.version>
```

Comment:
You are right, there is not need to update the core version just for this method.